modules/katello_repository.py

```diff
@@ -73,6 +73,14 @@
             - immediate
             - on_demand
         required: false
+    state:
+        description:
+        - State of the Global Parameter
+        default: true
+        choices:
+        - present
+        - lastest
+        - absent
```

Comment:
you already know, i believe. ;)

Reply:
So no latest... wth is present_with_defaults (in this context?) or should i drop it?

Reply:
It means 'create it if it was not there, leave it alone otherwise'.